graphql-server/imports/api/apollo/resolvers/Types/KeyPartner/index.js

```diff
@@ -47,5 +49,25 @@ export default {
 
       return byQuery.load({ documentId });
     },
+    nonConformities: async (root, args, context) => {
+      const { nonConformityIds = [] } = root;
+      const { loaders: { NonConformity: { byQuery } } } = context;
+
+      return byQuery.loadMany(map(nonConformityId => ({
```

Comment:
you should move this somewhere and reuse
```suggestion
      return byQuery.load({ _id: { $in: nonConformityIds }, isDeleted: false })
```